### ISSUE TITLE
Julia v1.6 not precompiling Stipple, throws error of "no method matching Base.TOMLCache()"

### ISSUE THREAD

**Issue by amckay1** [GenieFramework/Stipple.jl#44]
There is a closed issue with what seems to be this same issue (https://github.com/GenieFramework/Stipple.jl/issues/3) but thought a new issue might be more helpful since the bug is still alive it seems. Running Julia v1.6 and doing a simple `]add Stipple` or specifying `#master` or `v0.9.8` results in Stipple being unable to pass precompilation after `using Stipple` on my ubuntu machine.

**Comment by essenciary** [comment]
Thanks - not something Stipple specific. Seems to come from Revise within Genie: https://github.com/timholy/Revise.jl/issues/569 
What do you get for 
```julia
pkg> st Revise
```

**Comment by amckay1** [comment]
Thanks, that's interesting, `]st Revise` results in "No Matches". Out of curiosity, I added Revise (defaulted to v3.1.3 as noted in the link you provided) and rm/added Stipple, but got caught on the same error. 

Going to try and get Revise to v3.1.7 and try again but currently caught on some "Unsatisfiable requirements detected for package CUDA" shenanigans I'm going to have to sort out first I think.

**Comment by amckay1** [comment]
Okay, so Transformers.jl seemed to be holding back CUDA for Revise and once I installed Revise v3.1.7 then Stipple v0.9.9 precompiled just fine. Thanks for your help! You were definitely right about the Revise issue, hopefully that isn't too hard to untangle.

**Comment by essenciary** [comment]
Are you using all these packages in the same project? Maybe you can do `pkg> up Revise`?

**Comment by rben01** [comment]
I had the same issue, which I fixed by removing `DebugAdapter` (used by the Julia for VSCode project) from my environment. The issue was caused by `DebugAdapter` pinning a version of `JuliaInterpreter` that was itself pinning an old version of `Revise`. Only once `DebugAdapter` was removed could I update `Revise` to a version that fixed this (3.1.16).

**Comment by essenciary** [comment]
@rben01 Thanks for the updates. I'll close this.
